packages/tlschema/src/records/TLUser.ts

```diff
@@ -52,9 +52,12 @@ export const TLUser = createRecordType<TLUser>('user', {
 	let lang
 	if (typeof window !== 'undefined' && window.navigator) {
 		const availLocales = LANGUAGES.map(({ locale }) => locale) as string[]
-		lang = window.navigator.languages.find((lang) => {
-			return availLocales.indexOf(lang) > -1
-		})
+		lang =
+			availLocales.indexOf(window.navigator.language) > -1
+				? // use the navigator's preferred language (if we have one)
+				  window.navigator.language
+				: // otherwise, use the first available language
+				  window.navigator.languages.find((lang) => availLocales.indexOf(lang) > -1)
```

Comment:
I think was fine before @steveruizok see https://developer.mozilla.org/en-US/docs/Web/API/Navigator/languages

They are in order or preference anyway.

> The Navigator.languages read-only property returns an array of strings representing the user's preferred languages. The language is described using language tags according to [RFC 5646: Tags for Identifying Languages (also known as BCP 47)](https://datatracker.ietf.org/doc/html/rfc5646). In the returned array they are ordered by preference with the most preferred language first.

Reply:
I think this is essentially a no-op, unless I'm missing something.

Reply:
Ah good catch. I had a guy on Twitter asking why his language was set to French when he didn't have it set to French (though he was in France). This was the closest I could find to why that might be, though in that case it must be something else.

Reply:
Ok, there was a bug but wasn't it, updating OP